addons/hr_expense/models/account_payment.py

```diff
@@ -33,7 +33,8 @@ def _synchronize_from_moves(self, changed_fields):
 
     def _synchronize_to_moves(self, changed_fields):
         # EXTENDS account
-        if self.expense_sheet_id:
+        trigger_fields = list(self._get_trigger_fields_to_synchronize()) + ['ref', 'expense_sheet_id', 'payment_mode']
```

Comment:
When I said payment mode, I meant payment_method_line_id
There is no payment_mode on an account.payment
Nitpick, always use a set instead. No need to choose a slower way

Please test functionally your fixes and the requested changes
```suggestion
        trigger_fields = set(self._get_trigger_fields_to_synchronize()) | {'ref', 'expense_sheet_id', 'payment_method_line_id'}
```

Reply:
Sorry I left `payment_mode` there as a placeholder and then forgot about it :sweat_smile: